cves/kernel/CVE-2020-11494.yml

```diff
@@ -19,14 +19,14 @@ curated_instructions: |
   This will enable additional editorial checks on this file to make sure you
   fill everything out properly. If you are a student, we cannot accept your work
   as finished unless curated is properly updated.
-curation_level: 0
+curation_level: 2
 reported_instructions: |
   What date was the vulnerability reported to the security team? Look at the
   security bulletins and bug reports. It is not necessarily the same day that
   the CVE was created.  Leave blank if no date is given.
 
   Please enter your date in YYYY-MM-DD format.
-reported_date:
+reported_date: '2020-04-02'
```

Comment:
It seems that the fix was authored and reported on Apr 1, 2020 from the github commit.

Reply:
yes that's right but that is when the changes were committed. If you go to this website https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-11494 it shows when it was reported